conbench/templates/compare-list.html

```diff
@@ -55,59 +55,44 @@ <h5 class="card-title"></h5>
 </div>
 {% endif %}
 <p>
-  The <code>change</code> column below shows the direct comparison between matching benchmark results.
+  <a data-bs-toggle="collapse"
+     href="#collapseHelpblock"
+     role="button"
+     aria-expanded="false"
+     aria-controls="collapseHelpblock">toggle help for using the table below</a>
```

Comment:
Maybe you could make this a button? My eyes skipped right over this even when I was looking for it :)
<img width="446" alt="image" src="https://github.com/conbench/conbench/assets/16600275/2ab5ea3f-9442-4a84-b29c-42e4c79aa21c">

Reply:
> Maybe you could make this a button? My eyes skipped right over this even when I was looking for it :)

Hm. I will keep this for now and maybe iterate on it later. At first I had this as a button and it was wayyyy to prominent I thought. I think there's probably a middle ground where it's not what it's now, but also not a button.